src/TFLOrigins.tsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from "react"
+import { getOriginsData } from "../backend/fetchOriginsData"
+
+interface DisplayedOriginsDataType {
+    year: string,
+    text: string,
+    imgSource: string,
+    imgCaption: string,
+    source: string
+}
+
+function TFLOrigins() {
+
+    const initialDisplayedOriginsData = {
+        year: "",
+        text: "",
+        imgSource: "",
+        imgCaption: "",
+        source: ""
+    }
+
+    const [dataLoading, setDataLoading] = useState(true) 
+    const [originsData, setOriginsData] = useState([initialDisplayedOriginsData]);
+    const [originsPageNumber, setOriginsPageNumber] = useState(0)
+    const [displayedOriginsData, setDisplayedOriginsData] = useState<DisplayedOriginsDataType>(initialDisplayedOriginsData);
```

Comment:
I _think_ (but not certain) this could just be a standard variable
```suggestion
    const displayedOriginsData = originsData[originsPageNumber]
```